Automattic/wp-calypso
# Help Center: Get unread count when new message arrives

Related to #

## Proposed Changes

* Update hc unread conversations when:
	- users receive a message and the hc is minimized
	- conversation is opened

## Why are these changes being made?


* We failed to update the unread count in those cases

## Testing Instructions



* Checkout this PR
* Open the HC with `help-center-experience` flag
* Open any unread conversations to mark them as read
* Close the help center and send a message back from zendesk
* Check if the notifications icon is updated accordingly
* Open the help center and the unread conversation
* The icon should change to reflect that all messages have been read

## Pre-merge Checklist
